testing/checkin/runtime/async.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+const { op_async_barrier_create, op_async_barrier_await, op_async_yield } = Deno
+  .core
+  .ensureFastOps();
```

Comment:
Side issue: let's rename `Deno.core.ensureFastOps()` to `Deno.core.generateOpBindings()`?

Reply:
I think we should bite the bullet and do https://github.com/denoland/deno_core/issues/223 which gets rid of this method. I never really liked having to ask for ops this way.

What do you think?

Reply:
Yes, that sounds like a better solution